src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/ProviderRelationship/IProviderRelationshipsService.cs

```diff
@@ -7,6 +7,7 @@ namespace Esfa.Recruit.Vacancies.Client.Infrastructure.Services.ProviderRelation
     public interface IProviderRelationshipsService
     {
         Task<IEnumerable<EmployerInfo>> GetLegalEntitiesForProviderAsync(long ukprn);
+        Task<bool> ProviderHasPermissionAsync(long ukprn, string accountPublicHashedId, long legalEntityId);
```

Comment:
Consider renaming to `HasEmployerPermittedProviderAsync` or `HasProviderGotEmployersPermission`

Reply:
👍 I'll go with `HasProviderGotEmployersPermissionAsync`